extensions/cornerstone/src/initWADOImageLoader.js

```diff
@@ -83,6 +102,28 @@ export default function initWADOImageLoader(
   initWebWorkers(appConfig);
 }
 
+/*
+taken from
+https://hg.orthanc-server.com/orthanc-dicomweb/file/tip/Plugin/WadoRsRetrieveFrames.cpp
+*/
+const typeForTS = {
```

Comment:
Hard coding this prevents adding custom types here such as the single part, or adding priority.  Suggest dropping this table.